modules/raw-rsa-keyring-node/src/oaep_hash_supported.ts

```diff
@@ -32,10 +32,13 @@ export const oaepHashSupported = (function () {
      */
     return false
   } catch (ex) {
-    needs(
-      ex.code === 'ERR_OSSL_EVP_INVALID_DIGEST',
-      'Unexpected error testing oaepHash.'
-    )
-    return true
+    if (ex instanceof NotSupported) {
+      needs(
+        ex.code === 'ERR_OSSL_EVP_INVALID_DIGEST',
+        'Unexpected error testing oaepHash.'
+      )
+      return true
+    }
```

Comment:
@seebees this is where `NotSupported` is used in a `src` path instead of a test path.